### ISSUE TITLE
Add alt text to the alert icon

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#105756]
## Background

> “Alert” icon is missing meaningful text equivalent.
> Defect 1 | Critical | Created:
> WCAG 1.1.1 Non-text Content | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below.
> https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-contact -> 1 of 5 Accredited representative information.
> The “Alert” graphic is not equivalent to the information depicted visually. For example, as a JAWS user, for step “1 of 5 Accredited representative information”, when the user reads the content in the “Alert” section, the JAWS user dose not read the text equivalent of the icon before alerting the user to contact the accredited representative. (New: 02/24/2025).

![Image](https://github.com/user-attachments/assets/517b8ac1-7ce3-4178-8412-b67af1f95630)

## Tasks
- [ ] Add alt text to the alert icon, in this warning alert (no need to change the role)

## Acceptance Criteria
- [ ] The warning on the `/representative-contact` page is fully accessible to all users.
